index.js

```diff
@@ -142,16 +142,24 @@ const _onBeforeInput = shortcutsOfWindow => (e, input) => {
 };
 
 /**
-* Registers the shortcut `accelerator`on the BrowserWindow instance.
+ * Registers the shortcut `accelerator`on the BrowserWindow instance.
  * @param  {BrowserWindow} win - BrowserWindow instance to register.
  * This argument could be omitted, in this case the function register
  * the shortcut on all app windows.
- * @param  {String} accelerator - the shortcut to register
+ * @param  {String|Array<String>} accelerator - the shortcut to register
  * @param  {Function} callback    This function is called when the shortcut is pressed
  * and the window is focused and not minimized.
- * @return {Undefined}
+ * @return {void}
  */
 function register(win, accelerator, callback) {
+	if (Array.isArray(accelerator) === true) {
+		accelerator.forEach(accelerator => {
+			if (typeof accelerator === 'string') {
+				register(win, accelerator, callback);
+			}
+		});
+		return;
+	}
```

Comment:
`win`  argument, is optional, you should move this block after the part 
```js
if (typeof callback === 'undefined') {
		wc = ANY_WINDOW;
		callback = accelerator;
		accelerator = win;
	} else {
		wc = win.webContents;
	}
```

Reply:
I'm trying to overload **register** method, to avoid side effects.
Initial idea was to rename existing register to _register and create new function register
```javascript
function register(win, accelerator, callback) {
	if (Array.isArray(accelerator) === true) {
		accelerator.forEach(accelerator => {
			if (typeof accelerator === 'string') {
				_register(win, accelerator, callback);
			}
		});
		return;
	} else {
		_register(win, accelerator, callback);
	}
function _register(win, accelerator, callback) {
	// current implementation of register
}
```
but I think it's too much

Reply:
> but I think it's too much

I think too. But if you just move the block I think it's fine. It also has the advantage that shortcut array could also be used to register shortcuts for any window of the app.

Reply:
Agree, PR updated.